.github/workflows/junit5-build.yml

```diff
@@ -34,3 +34,30 @@ jobs:
         with:
           arguments: --refresh-dependencies --stacktrace --scan clean build -PjunitMinorVersion=${{ matrix.junit5Minor }}
           wrapper-cache-enabled: false
+          
+  gradle:
```

Comment:
i will remove this and exclude it - but i thought it is also interesting to run at least once